bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -150,65 +148,76 @@ public static String executeUrl(String httpMethod, String url, Properties httpHe
             String contentType, int timeout, String proxyHost, Integer proxyPort, String proxyUser,
             String proxyPassword, String nonProxyHosts) {
 
-        HttpClient client = new HttpClient();
+        startHttpClient(client);
 
         // only configure a proxy if a host is provided
         if (StringUtils.isNotBlank(proxyHost) && proxyPort != null && shouldUseProxy(url, nonProxyHosts)) {
-            client.getHostConfiguration().setProxy(proxyHost, proxyPort);
-            if (StringUtils.isNotBlank(proxyUser)) {
-                client.getState().setProxyCredentials(AuthScope.ANY,
-                        new UsernamePasswordCredentials(proxyUser, proxyPassword));
+            AuthenticationStore authStore = client.getAuthenticationStore();
+            ProxyConfiguration proxyConfig = client.getProxyConfiguration();
+
+            // clear proxy list from previous requests
+            List<Proxy> proxies = proxyConfig.getProxies();
+            if (!proxies.isEmpty()) {
+                proxies.clear();
             }
+
+            HttpProxy proxy = new HttpProxy(proxyHost, proxyPort);
+            proxies.add(proxy);
+
+            authStore.addAuthentication(new BasicAuthentication(proxy.getURI(), null, proxyUser, proxyPassword));
```

Comment:
I have asked on the mailing list.
I have found that there is an [update ](http://www.eclipse.org/jetty/documentation/current/http-client-authentication.html)in the jetty documentation, since the last time, I have visited it.
It is explained how we can make a preemptive authentication without manually creating an "Authenticaiton" header:
```
AuthenticationStore auth = httpClient.getAuthenticationStore();
URI uri = URI.create("http://domain.com/secure");
auth.addAuthenticationResult(new BasicAuthentication.BasicResult(uri, "username", "password"));
```
The same can be used for  [proxy authentication](http://www.eclipse.org/jetty/documentation/current/http-client-proxy.html#http-client-proxy-authentication).
Maybe it is better to make a new commit with the changes.